crates/scroll/alloy/network/src/lib.rs

```diff
@@ -228,3 +231,13 @@ impl NetworkWallet<Scroll> for EthereumWallet {
         })
     }
 }
+
+impl RecommendedFillers for Scroll {
+    // TODO: We don't have a BlobGasFiller due to trait constraints, is this needed in the absence
```

Comment:
Do we need this comment? I don't think blob gas filler is important for us right?